src/styles/blocks/main-block.scss

```diff
@@ -0,0 +1,7 @@
+.main {
+
+  &__catalog {
+    margin: 0 auto;
+    padding: 50px 0 72px;
```

Comment:
set horizontal padding to `40px` and increase catalog width (1024px should be enough) - it will fix tests